shared/src/main/java/com/couchbase/lite/Database.java

```diff
@@ -489,27 +487,26 @@ protected void finalize() throws Throwable {
     // Package level access
     //---------------------------------------------
 
-    /* package */ ConflictResolver getConflictResolver() {
+    ConflictResolver getConflictResolver() {
         return config != null ? config.getConflictResolver() : null;
     }
 
     // Instead of clone()
-    /* package */ Database copy() {
+    Database copy() throws CouchbaseLiteException {
         return new Database(this.name, this.config);
     }
 
     //////// DATABASES:
 
     void mustBeOpen() {
         if (c4db == null)
-            throw new IllegalStateException("Database is not open.");
+            throw new CouchbaseLiteRuntimeException("Database is not open.");
```

Comment:
I think we should throw IllegalStateException instead of CouchbaseLiteRuntimeException as it's self described that this is in an inconsistent or illegal state that users shouldn't do. CouchbaseLiteRuntimeException should be something that LC might thrown but not expected?